FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Multitask/core_mqtt_config.h

```diff
@@ -75,6 +74,34 @@ extern void vLoggingPrintf( const char * pcFormatString,
  * macro sets the limit on how many simultaneous PUBLISH states an MQTT
  * context maintains.
  */
-#define MQTT_STATE_ARRAY_MAX_COUNT    20U
+#define MQTT_STATE_ARRAY_MAX_COUNT              20U
+
```

Comment:
A comment marker to clearly differentiate between the MQTT Agent configs and coreMQTT configs may be helpful
```suggestion
/********************************** coreMQTT Agent Configurations *****************************/
```